content/workspace/openbb-copilot.md

```diff
@@ -12,336 +12,616 @@ keywords:
+
+- **Body:** The primary interaction area where conversations unfold. This section displays the Copilot's responses, step-by-step reasoning processes, generated artifacts (charts, tables, code), and any intermediate results. The body automatically scrolls and organizes content chronologically, making it easy to follow complex multi-step analyses. When hovering on messages it also displays the datetime associated with these.
+
+- **Footer:** This is where the data and widget management occurs for what will be used as context for the agent. This is also where the user can add their own custom AI agents or access their prompt library.
+
+### Full AI-chat mode
 
-By utilizing all of these sources of data, OpenBB Copilot can assist you in
-performing a wide variety of research and analysis tasks.
+The Copilot panel offers adaptive sizing to match your workflow needs.
 
-<div style={{display: 'flex', justifyContent: 'center'}}>
-<img src="https://openbb-assets.s3.us-east-1.amazonaws.com/docs/copilot/ada_opened.png" alt="opened openbb copilot" width="70%" height="70%" />
+In side-panel mode, it provides quick access for simple queries while maintaining your primary workspace view.
+
+<div
+  style={{ display: "flex", justifyContent: "center", alignItems: "center" }}
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/1f9ba730-2516-4eac-b7c9-ca6e9382ba78.png"
+    width="100%"
+  />
 </div>
 
-## Understanding Chats
+When expanded to full-screen mode, you gain maximum real estate for complex conversations, detailed reasoning steps, and large artifacts like comprehensive charts or extensive data tables.
 
-OpenBB Copilot is a chat-based assistant that leverages multiple LLMs (Large
-Language Models) under-the-hood to understand your queries and provide answers.
+<div
+  style={{ display: "flex", justifyContent: "center", alignItems: "center" }}
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/7aa420b1-01b8-4785-b2d9-588ddd414842.png"
+    width="100%"
+  />
+</div>
 
-Since OpenBB Copilot is a chat-based assistant, it maintains a conversation history
-to provide contextual responses. This means you can have natural back-and-forth
-discussions, ask follow-up questions, and refine your queries based on previous
-responses. The conversation history helps OpenBB Copilot understand the context of
-your questions and provide more relevant and personalized assistance throughout
-your interaction.
+This flexibility allows seamless transitions between quick consultations and deep analytical sessions without losing context or interrupting your workflow.
 
-For example, if you ask about the price-to-earnings (P/E) ratio, and then follow
-up by asking about other important financial ratios, OpenBB Copilot will
-understand the context and suggest relevant ratios:
+And of course you are capable of hiding the AI agent if only utilizing the workspace.
```

Comment:
Suggest changing "if only interacting woth workspace" to something like "if you want to work solely with the dashboard"